lib/wings/model_transformer.rb

```diff
@@ -81,6 +82,8 @@ def self.convert_class_name_to_valkyrie_resource_class(class_name)
     #   results in long methods
     def self.to_valkyrie_resource_class(klass:)
       Class.new(ActiveFedoraResource) do
+        include Wings::PcdmValkyrieBehavior
```

Comment:
This brings in the replacement code for #objects into the newly create valkyrie resource class.